dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/JdbcEventStore.java

```diff
@@ -928,6 +920,32 @@ private String getEventSelectQuery(
         .toString();
   }
 
+  private String addOrderFieldsToSelectClause(List<Order> orders) {
```

Comment:
```suggestion
  private String getOrderFieldsToSelectClause(List<Order> orders) {
```

Its a pure function `List<Order> -> String` that does not add anything. There might be a better name than my suggestion but `add` is misleading.